VideoTransformers/CHANGELOG.md

```diff
@@ -0,0 +1,32 @@
+# Video Transformers Changelog
+
+All notable changes to this project will be documented in this file.
+
+## 2.25.2
+
+### Added
+
+- Support pre-built transformers in the Vonage Media Processor library or create your own custom video transformer to apply to published video. NVIDIA GPUs are recommended for optimal performance.
+
+### Known issues
+
+- When using Vonage's Background Blur, the ML model is not present on the dll. It is required to be added manually: The steps are the following:
```

Comment:
I suppose this is still true, right?
I imagine this will it be updated once the nuget updates are committed, right?

Reply:
For 2.25.2 it is still true indeed. Afterwards when 2.25.3 is created, we will document that the issue was solved